components/service/experiments/src/test/java/mozilla/components/service/experiments/ExperimentsTest.kt

```diff
@@ -322,6 +324,23 @@ class ExperimentsTest {
         assertEquals("second-id", experiments.activeExperiment!!.experiment.id)
     }
 
+    @Test
+    fun `experiment enrollment recorded in Glean`() {
+        resetExperiments()
+        experiments.initialize(mockContext, configuration)
+        experiments.experimentsResult = ExperimentsSnapshot(experimentsList, null)
+        experiments.findAndStartActiveExperiment()
```

Comment:
I'd add `assertFalse(Glean.testIsExperimentActive("second-id"))` before this, just to make sure we're in a consistent state.

Reply:
👍